qa-heal.planx-pla.net/metadata/aggregate_config.json

```diff
@@ -1,12 +1,13 @@
 {
 	"gen3_commons": {
 		"HEAL": {
-			"mds_url": "https://qa-heal.planx-pla.net",
-			"commons_url": "qa-heal.planx-pla.net"
+			"mds_url": "https://preprod.healdata.org",
+			"commons_url": "preprod.healdata.org"
```

Comment:
```suggestion
			"commons_url": "qa-heal.planx-pla.net"
```

Reply:
@xritter1  I changed the mds_url of preprod back to qa-heal -- sorry about that!